index.d.ts

```diff
@@ -0,0 +1,432 @@
+// Type definitions for ldclient-node
+
+/**
+ * The LaunchDarkly Node.js client interface.
+ *
+ * Documentation: http://docs.launchdarkly.com/docs/node-sdk-reference
+ */
+
+declare module 'ldclient-node' {
+  /**
+   * The LaunchDarkly static global.
+   */
+  export function init(key: string, user: LDUser, options?: LDOptions): LDClient;
```

Comment:
Great catch, that definitely shouldn't be there.

Will patch in a few, thanks @natebrennand